packages/salesforcedx-vscode-core/src/orgBrowser/orgMetadata.ts

```diff
@@ -88,34 +88,40 @@ export async function forceDescribeMetadata(outputPath?: string) {
 }
 
 export async function getMetadataTypesPath(): Promise<string | undefined> {
-  if (hasRootWorkspace()) {
-    const workspaceRootPath = getRootWorkspacePath();
-    const defaultUsernameOrAlias = await OrgAuthInfo.getDefaultUsernameOrAlias(
-      false
-    );
-    const defaultUsernameIsSet = typeof defaultUsernameOrAlias !== 'undefined';
-
-    if (defaultUsernameIsSet) {
-      const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias!);
-      const metadataTypesPath = path.join(
-        workspaceRootPath,
-        '.sfdx',
-        'orgs',
-        username,
-        'metadata',
-        'metadataTypes.json'
-      );
-      return metadataTypesPath;
-    } else {
-      const err = nls.localize('error_no_default_username');
-      telemetryService.sendError(err);
-      throw new Error(err);
-    }
-  } else {
+  if (!hasRootWorkspace()) {
```

Comment:
Same functionality, I just refactored it in order to have less if/else's and error out as soon as possible.